### PR TITLE
Docker for MySQL configuration added

### PR DESCRIPTION
**Pull request by mateuszkp96**

### CODE REVIEW

File: db/scripts/schema.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+DROP SCHEMA IF EXISTS `rso`;
+CREATE SCHEMA IF NOT EXISTS `rso`;
+USE rso;
+
+CREATE TABLE IF NOT EXISTS client
+(
+    id    BIGINT(20)  NOT NULL AUTO_INCREMENT,
```
Comment (nowpawel96) [inline db/scripts/schema.sql]: With JPA it is better to use sequences for IDs

File: db/run.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+name='singleMySql'
```
Comment (nowpawel96) [inline db/run.sh]: It's database name or just a program name?

Comment (mateuszkp96) [reply db/run.sh]: Name of container

File: db/scripts/schema.sql (hunk 1 above)
Comment (mateuszkp96) [reply db/scripts/schema.sql]: It's only example how to load schema and data to container. Data scripts will be replaced for sure.

**Pull request merged**